app/controllers/donuts_controller.rb

```diff
@@ -2,4 +2,8 @@ class DonutsController < ApplicationController
   def index
     @donuts = Donut.all
   end
+
+  def show
+    @donut = Donut.find(params[:id])
+  end
```

Comment:
You shouldnt need line 7 since this is in React!

Reply:
donezo!